src/common/containers/content/pages/widgets/data_tables/table_entries/ProjectsDataTableEntry.jsx

```diff
@@ -9,8 +9,12 @@ import { connect } from 'react-redux';
 import ProjectsDataTableEntry from '../../../../../../components/content/widgets/data_tables/table_entries/ProjectsDataTableEntry'; // eslint-disable-line max-len
 
 const mapStateToProps = (state) => {
+    const { basePath } = state;
+    const { user } = state.user;
```

Comment:
Your link pointed to users.
What I'm getting at is if you're guarding against issues when the "state.user" is the initial value.